Description: Correct PHP5 search
 The php5 executable is not required to determine
 the version. Include dirs and version can be
 constructed using php-config.
Author: Paul Klos <kolab@klos2day.nl>
Last-Update: 2013-04-13
Bug: https://issues.kolab.org/show_bug.cgi?id=1753

--- a/src/php/CMakeLists.txt
+++ b/src/php/CMakeLists.txt
@@ -22,7 +22,6 @@
 
 # Try to find PHP5
 find_path(PHP_INCLUDE_DIR NAMES main/php.h PATH_SUFFIXES php php5)
-find_program(PHP_EXECUTABLE NAMES php)
 
 # Libkolabxml needs PHP >= 5.3
 set(PHP_MIN_VERSION 50300)
@@ -41,7 +40,7 @@
   endif()
 endif()
 
-if(PHP_INCLUDE_DIR AND PHP_EXECUTABLE)
+if(PHP_INCLUDE_DIR AND PHP_CONFIG_EXECUTABLE)
   file(READ ${PHP_INCLUDE_DIR}/main/php_version.h PHP_VERSION_CONTENT)
   string(REGEX MATCH "#define PHP_VERSION_ID[ ]*[0-9]*\n" _PHP_VERSION_ID_MATCH ${PHP_VERSION_CONTENT})
   if(_PHP_VERSION_ID_MATCH)
